core/metacling/src/TCling.cxx

```diff
@@ -121,6 +121,7 @@ clang/LLVM technology.
 #include "llvm/Support/DynamicLibrary.h"
 #include "llvm/Support/raw_ostream.h"
 #include "llvm/Support/Path.h"
+#include "llvm/Support/Program.h"
```

Comment:
```suggestion
```

Reply:
We can't delete this. `llvm::sys::writeFileWithEncoding` lives in this header.

Reply:
Ah, ok, I missed that.